cadasta/api/organization.py

```diff
@@ -0,0 +1,57 @@
+# coding=utf-8
+"""
+Cadasta project - **Organization api.**
+
+.. note:: This program is free software; you can redistribute it and/or modify
+     it under the terms of the GNU General Public License as published by
+     the Free Software Foundation; either version 2 of the License, or
+     (at your option) any later version.
+
+"""
+
+from qgis.PyQt.QtCore import QCoreApplication
+from cadasta.mixin.network_mixin import NetworkMixin
+
+__author__ = 'dimas@kartoza.com'
+__revision__ = '$Format:%H$'
+__date__ = '26/02/2014'
+__copyright__ = 'Copyright 2016, Cadasta'
+
+
+class Organization(object):
+    """Organization api class"""
+
+    api_url = 'https://demo.cadasta.org/api/v1/organizations/'
+
+    def get_api(self, network):
+        """
+        Execute api get call
```

Comment:
Please follow this link for the docstring : https://github.com/inasafe/inasafe/wiki/Coding-Standards#doc-strings-and-comments.
The example is correct.
* A full sentence with a dot.
* New line between parameters
* Add `:rtype`